pkg/auth/cli.go

```diff
@@ -25,6 +25,7 @@ type LoginOptions struct {
 	Stdin                     io.Reader // set to os.Stdin
 	Stdout                    io.Writer // set to os.Stdout
 	AcceptUnspecifiedRegistry bool      // set to true if allows login with unspecified registry
+	AcceptRepositories        bool      // set to true to allow repositories rather than just registries
```

Comment:
(Also logout)

Shouldn’t this be a user option (also in `GetLoginFlags`/`GetLogoutFlags`)? How would the caller like Podman decide what to set this to?

Reply:
I think so, added it to them as well.